inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/inner/client/InlongStreamClient.java

```diff
@@ -295,11 +300,15 @@ public List<StreamField> parseFields(String method, String statement) {
         return parseFields(request);
     }
 
-    public List<BriefMQMessage> queryMessage(String groupId, String streamId, Integer messageCount) {
-        Preconditions.expectNotBlank(groupId, ErrorCodeEnum.GROUP_ID_IS_EMPTY);
-        Preconditions.expectNotBlank(streamId, ErrorCodeEnum.STREAM_ID_IS_EMPTY);
+    public List<BriefMQMessage> queryMessage(QueryMessageRequest request) {
+        Preconditions.expectNotBlank(request.getGroupId(), ErrorCodeEnum.GROUP_ID_IS_EMPTY);
+        Preconditions.expectNotBlank(request.getStreamId(), ErrorCodeEnum.STREAM_ID_IS_EMPTY);
+        Map<String, Object> requestMap = JsonUtils.OBJECT_MAPPER.convertValue(request,
```

Comment:
JsonUtils.parseObject

Reply:
The purpose here is to convert the object into a map, which `JsonUtils.parseObject` cannot achieve.